packages/edit-site/src/components/header/index.js

```diff
@@ -109,22 +114,26 @@ export default function Header( { openEntitiesSavedStates } ) {
 			</div>
 
 			<div className="edit-site-header_center">
-				<DocumentActions
-					entityTitle={ title }
-					entityLabel={
-						templateType === 'wp_template'
-							? 'template'
-							: 'template part'
-					}
-				>
-					{ templateType === 'wp_template' &&
-						( ( { onClose } ) => (
-							<TemplateDetails
-								template={ template }
-								onClose={ onClose }
-							/>
-						) ) }
-				</DocumentActions>
+				{ 'wp_template' === templateType && (
+					<DocumentActions
+						entityTitle={ entityTitle }
+						entityLabel="template"
+					>
+						{ templateType === 'wp_template' &&
+							( ( { onClose } ) => (
+								<TemplateDetails
+									template={ template }
+									onClose={ onClose }
+								/>
+							) ) }
+					</DocumentActions>
+				) }
+				{ 'wp_template_part' === templateType && (
+					<DocumentActions
+						entityTitle={ entityTitle }
+						entityLabel="template part"
+					/>
+				) }
```

Comment:
We have Yoda and non-yoda conditions here. Can we make them consistent? Sorry for nitpicking :D

Reply:
😱 You want me to fix a pre-existing non-yoda while tests are passing?? 😆